rl4co/models/nn/improvement_attention.py

```diff
@@ -0,0 +1,225 @@
+import math
+
+from typing import Callable, Tuple, Union
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from rl4co.models.nn.ops import Normalization
+from rl4co.utils.pylogger import get_pylogger
+
+log = get_pylogger(__name__)
+
+
+class mySequential(nn.Sequential):
```

Comment:
[Minor] This is a nice idea! This could also be useful for other tasks. So maybe we can give this a more readable name like `AdaptiveSequential ` and add it to nn.ops

Reply:
Yeah, I have refactored to nn.ops